boards/OPENMV3/omv_boardconfig.h

```diff
@@ -148,6 +148,7 @@
 #define OMV_CSI_TIM_CLK_SLEEP_DISABLE()       __TIM1_CLK_SLEEP_DISABLE()
 #define OMV_CSI_TIM_PCLK_FREQ()               HAL_RCC_GetPCLK2Freq()
 #define OMV_CSI_HW_CROP_ENABLE                (1)
+#define OMV_OV7725_CLK_FREQ                   OMV_CSI_CLK_FREQUENCY
```

Comment:
This belongs with the other OV7725 settings.

Reply:
Done